mount/infoParser.go

```diff
@@ -159,13 +127,23 @@ func (mi *mountInfoParser) parseData(data []byte) error {
 
 		mi.mpInfo[parsedMounts.MountPoint] = parsedMounts
 		mi.idInfo[parsedMounts.MountID] = parsedMounts
+
+		// Optimization needed here -- this logic makes sense only for remount ops.
```

Comment:
This comment is a bit confusing; at this level nothing is known about remount ops.

Reply:
Yep, that's an old comment I forgot to remove.